src/client/modules/my/rawResultsTable/rawResultsTable.html

```diff
@@ -4,30 +4,24 @@
 
     <thead>
       <tr class="slds-line-height_reset">
-        <th>Player</th>
-        <th>Games Played</th>
-        <th>Percentage of Games Played</th>
-        <th>Average Points per Game</th>
-        <th>Total Points</th>
-        <th>Number of First Place Finishes</th>
-        <th>First Place Percentage</th>
+        <th>Date</th>
+        <th>Name</th>
+        <th>Ranking</th>
```

Comment:
Technically, in Matt's raw results he had Date, Ranking, Name, Victory Points, rather than Date, Name, Ranking, Victory Points. I know this is being nit-picky, but if you just switch ranking and name it will be exactly the same